app/scripts/controllers/classify.js

```diff
@@ -31,7 +29,7 @@ angular.module('moodCatApp')
                         $scope.songs = response;
                         $timeout.cancel(timeout);
                         currentSongService.stop();
-                        $scope.activeSong = null;
+                        $scope.song = null;
```

Comment:
This should be $rootScope as per https://github.com/MoodCat/MoodCat.me-Frontend/pull/72/files#diff-89093d224f00acffebc38bf41e7aa977R41